src/components/NoteDialogComponents/NoteDialogHeader.tsx

```diff
@@ -2,7 +2,7 @@ import {FC} from "react";
 import "./NoteDialogHeader.scss";
 
 type NoteDialogHeaderProps = {
-  columnName: string;
+  columnName?: string;
```

Comment:
it was to prevent errors when the column couldn't be found because the note was deleted. But I changed it so that an empty string will be passed as prop in that case